zipkin-cassandra/src/main/scala/com/twitter/zipkin/storage/cassandra/CassandraDependencyStore.scala

```diff
@@ -0,0 +1,39 @@
+package com.twitter.zipkin.storage.cassandra
+
+import com.twitter.util.TimeConversions._
+import com.twitter.util._
+import com.twitter.zipkin.common.Dependencies
+import com.twitter.zipkin.conversions.thrift._
+import com.twitter.zipkin.storage.DependencyStore
+import com.twitter.zipkin.thriftscala.{Dependencies => ThriftDependencies}
+import org.twitter.zipkin.storage.cassandra.Repository
+
+import scala.collection.JavaConverters._
+
+/**
+ * This implementation of DependencyStore assumes that the job aggregating dependencies
```

Comment:
this thing was the reason it took me so long to finish this. eventually, I figured out why everything was flooring to a day!